packages/components/src/dialog/style.scss

```diff
@@ -1,5 +1,5 @@
-@import '../styles/breakpoints';
-
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
```

Comment:
If both of these imports are necessary for using the Gutenberg breakpoints, it might be a good thing to add to the documentation in the [other PR](https://github.com/Automattic/wp-calypso/pull/41984), even if just as part of a short guide of "How to convert `breakpoint-deprecated` to Gutenberg `break-*`".

Reply:
You could also add them in a separate file and include only that.

Reply:
BTW I can't remember if there was reason not to import breakpoints in mixins file in core base-styles, but that's also worth considering.

Reply:
As an aside, the `mixins` import is not _always_ needed, especially if we compare with the Calypso breakpoint implementation.

In Gutenberg we do "mobile first" and only go up from there, while Calypso also permits going "down".
In other words, the Calypso `breakpoint()` lets you target both `min-width` and `max-width`, whereas Gutenberg only `min-width`.

When replacing a Calypso `breakpoint( '>480px' )`, we would use Gutenberg's `break-mobile()` mixin, thus requiring both `mixins` and `breakpoints` imports.
When replacing a Calypso `breakpoint( '<480px' )`, we just need to import Gutenberg's `breakpoints`, and write `@media ( max-width: $break-mobile )`.